src/pages/resultsView/coExpression/styles.scss

```diff
@@ -0,0 +1,31 @@
+.coexpression-plot-data-tooltip {
```

Comment:
might as well nest all this inside of one tab specific class

Reply:
no bc the div style needs to apply to the div child, max-width to the parent

Reply:
good idea. should I remove the `coexpression-plot` prefix or just keep it too?